api/src/main/scala/uk/ac/wellcome/platform/api/models/DisplayLicense.scala

```diff
@@ -0,0 +1,27 @@
+package uk.ac.wellcome.platform.api.models
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.swagger.annotations.{ApiModel, ApiModelProperty}
+
+@ApiModel(
+  value = "License",
+  description =
+    "The specific license under which the work in question is released to the public - for example, one of the forms of Creative Commons - if it is a precise license to which a link can be made."
```

Comment:
That would be great and something I'm sure @alicefuzier would love to help with 😄